Tests/ApolloServerIntegrationTests/UploadTests.swift

```diff
@@ -6,21 +6,33 @@ import StarWarsAPI
 
 class UploadTests: XCTestCase {
 
-  let uploadClientURL = TestServerURL.uploadServer.url
+  static let uploadClientURL = TestServerURL.uploadServer.url
```

Comment:
I can at least see the rationale for this change a bit better, since it's utterly unchanging throughout the test suite, but I do think this may be a premature optimization since the instance is created and destroyed with every test since it's a `let`, so it's tied to the lifetime of the test.